src/Parsers/GlobeCoordinateParser.php

```diff
@@ -38,9 +38,8 @@ class GlobeCoordinateParser implements ValueParser {
 
 	public function __construct( ?ParserOptions $options = null ) {
 		$this->options = $options ?: new ParserOptions();
```

Comment:
Why not:

```
$this->options = ($options ?: new ParserOptions())
   ->withDefaultOption(...)
   ->withDefaultOption(...);
```

Reply:
Also works, but I couldn't be bothered in this case